environment

```diff
@@ -52,42 +50,54 @@ EXPORT_ENV_VARS_TO_LAMBDA_ARRAY=(
 set -a
 EXPORT_ENV_VARS_TO_LAMBDA=${EXPORT_ENV_VARS_TO_LAMBDA_ARRAY[*]}
 DSS_DEPLOYMENT_STAGE=dev
+STAGE="${DSS_DEPLOYMENT_STAGE}"
```

Comment:
was there a reason for adding in another "stage" var?

Reply:
yeah, because it's easier to type than DSS_DEPLOYMENT_STAGE

:P

will fix

Reply:
fixed with 24adad4